scripts/migration/migrate_ultimate_parent_node.py

```diff
@@ -0,0 +1,76 @@
+"""
+This will add an ultimate_parent field to all nodes.
+Ultimate_parent will be the primary key of the originating parent node
```

Comment:
Please update the name of this script and update all references to `ultimate_parent`.